requirements.txt

```diff
@@ -1,11 +1,11 @@
 # Core project/common requirements
 celery==3.1.18
-Django==1.8.3
+Django==1.8.4
 django-appconf==1.0.1
-django-bootstrap3==6.1.0
+django-bootstrap3==6.2.1
 django-compressor==1.5
 django-haystack==2.4.0
-djangorestframework==3.1.3
+djangorestframework==3.2.2
```

Comment:
Looks fine to me. Unfortunately the `RequiredBooleanField` workaround we have is still necessary